packages/ens-deployments/src/lib/types.ts

```diff
@@ -52,35 +47,37 @@ export type SubregistryContractConfig =
     };
 
 /**
- * Encodes the deployment of a subregistry, including the target chain and contracts.
+ * A DeploymentDatasource describes a set of contracts on a given chain that extend the ENS root.
```

Comment:
```suggestion
 * A DeploymentDatasource describes a set of contracts on a given chain that interact with the ENS protocol.
```

Suggesting this change because what about the data source that includes the ENS root itself? That data source doesn't extend the ENS root. Instead it includes it. So trying to make the language more general.